app/services/search/postgres/reading_list.rb

```diff
@@ -72,9 +72,11 @@ def self.find_articles(user:, term:, statuses:, tags:, page:, per_page:)
           .select(*REACTION_ATTRIBUTES)
           .to_sql
 
-        relation = Article.joins("INNER JOIN (#{reaction_query_sql}) reactions ON reactions.reactable_id = articles.id")
+        relation = ::Article.joins(
```

Comment:
changed to `::Article` here, otherwise it would pick up `Search::Postgres::Article` being in the same module